linux_os/guide/services/ssh/ssh_server/sshd_set_keepalive/tests/correct_value.pass.sh

```diff
@@ -1,9 +0,0 @@
-#!/bin/bash
```

Comment:
I would not remove this scenario, based on https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/var_sshd_set_keepalive.var#L19 the default value should be `0` and it should work with the `(all)` profile, so let's just remove the `profiles` metadata.

Reply:
Sure, I agree with it.